clang/lib/CIR/CodeGen/CIRGenExpr.cpp

```diff
@@ -257,9 +260,8 @@ LValue CIRGenFunction::buildLValueForBitField(LValue base,
     llvm_unreachable("NYI");
   }
 
-  Address Addr = getAddrOfField(base, field, Idx);
-
   const unsigned SS = useVolatile ? info.VolatileStorageSize : info.StorageSize;
+  Address Addr = getAddrOfBitFieldStorage(base, field, Idx, SS);
```

Comment:
What's LLVM codegen doing here? Does it call `getAddrOfBitFieldStorage` or `getAddrOfField`?

Reply:
there is no LLVM codegen here.

Reply:
I mean, we follow the principle of using the same skeleton as `clang/lib/CodeGen`, at least to the extend of what's possible.

I don't see anything similar to `getAddrOfBitFieldStorage` or `getAddrOfField` in `clang/lib/CodeGen/*`, and I want to understand what's the rationale there. Are those names introduced for specific functionality you refactor into helpers or are they complete new names for something that already existed in `clang/lib/CodeGen`? If so, I'm trying to understand why can't you just `s/Emit/build` and call it a day?